packages/dbml-core/src/parse/mysql/parser.pegjs

```diff
@@ -123,27 +123,42 @@ PKSyntax = _ primary_key _ "(" _ names:ListOfNames _ ")"
 {return names}
 
 // FKSyntax: Support "FOREIGN KEY (field[, field]*) REFERENCES `table`(field[, field]*)"
-FKSyntax = _ ("CONSTRAINT"i _ name)? _ foreign_key _ 
+FKSyntax = _ constraint:("CONSTRAINT"i _ name)? _ foreign_key _ 
 	"(" _ fields:ListOfNames _ ")" _
 	references _ table2:table_name _ "(" _ fields2:ListOfNames _ ")" _
-	("ON"i _ ("DELETE"i/"UPDATE"i) _ references_options)?
+	onDelete:FKOnDelete? _
+	onUpdate:FKOnUpdate?
```

Comment:
I've checked with MySQL and Postgres that the order of `ON DELETE action` and `ON UPDATE action` doesn't matter. But if you parsed like above, `ON DELETE action` must always be placed before `ON UPDATE action`. Pls help fix this.

Reply:
@phuongduyphan This should be fixed now.